### PR TITLE
[G4VECGEOM] Update Geant4 to 11.1.ref07

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test

File: geant4.spec [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-#%define branch cms/v%{realversion}
-#%define github_user cms-externals
 %define tag cf4c6712c60cdf2b4d8799e34e176917c2de358d
-%define branch master
-%define github_user Geant4
+%define branch cms/v%{realversion}
+%define github_user cms-externals
+#%define tag cf4c6712c60cdf2b4d8799e34e176917c2de358d
+#%define branch master
+#%define github_user Geant4
```
Comment (smuzaffar) [inline geant4.spec]: just update the value (tag and branch) without keeping the old values in commented out code

Comment (iarspider) [reply geant4.spec]: I will clean it up once tests are done

**Comment by iarspider** [comment]
please test

**Comment by civanch** [comment]
@iarspider , there are two set of G4 libraries: geant4core and geant4vis. This problem means that the second is not compiled properly or we need to move G4persistency from vis to core, because cmake of Geant4 were changed. Most likely we have to review both spec files

**Comment by iarspider** [comment]
please test

**Pull request merged**